test/end-to-end/util/LinkCreationProcedure.ts

```diff
@@ -31,13 +31,15 @@ const fetchLink = async (url, numberOfFetches) => {
     return res.ok
   }
 
-  Promise.all(new Array(numberOfFetches).fill(get(url))).then(
-    (values: boolean[]) => {
-      console.log(
-        `Url: ${url} was fetched ${values.filter(Boolean).length} times`,
-      )
-    },
-  )
+  const fetchArray: Promise<boolean>[] = []
```

Comment:
Context behind the bug: The way this array was populated only made the `fetch` call once despite the `numberOfFetches` specified. I suspect that the underlying implementation of this method creates a single instance of `get(url)` and populates the array with the same object references, hence only needing a single promise to be resolved/